product-service/services/productService.js

```diff
@@ -0,0 +1,11 @@
+import productsData from "../data/products.json";
+
+const getProducts = async () => productsData;
```

Comment:
[Proposal]

As the alternative to this syntax we can use export on functions level

export const getProducts = ...
export const getProductById = ...

It will produce the same result as export default { getProducts, getProductById } with the less lines of code.